secure_message/repository/database.py

```diff
@@ -49,6 +51,8 @@ def __init__(self, msg_id="", subject="", body="", thread_id="", collection_case
         self.survey = survey
         self.collection_exercise = collection_exercise
         self.from_internal = from_internal
+        self.sent_at = datetime.now(timezone.utc)
```

Comment:
This can be done as part of the column definition:
```
sent_at = Column('sent_at', DateTime, default=datetime.datetime.utcnow)
```

http://docs.sqlalchemy.org/en/latest/core/defaults.html

Reply:
Done, much tidier ^^